homework/yaryna/count_holes_hw_6.py

```diff
@@ -0,0 +1,20 @@
+holes = {"0":1,"4":1,"6":1,"8":2,"9":1}
+
+string = str(input("Enter sequence:"))
+
+def count_holes(string):
+    string = string.replace("-","")
+    #if isinstance(string,int) == True  or string.isdigit()==True:
+    if string.isdigit()==True:
```

Comment:
you do not need to compare this expression twice. It can be just `if string.isdigit():`